src/content/docs/style-guide/writing-docs/writer-workflow/swiftype-search.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Swiftype search
+topics:
+  - Tech writer style guide
+  - Processes and procedures
+---
+
+
+import styleguideNewRelicUiQuestionMark from 'images/style-guide_screenshot-crop_new-relic-ui-question-mark.png'
+
+import styleguideUiSearchString from 'images/style-guide_screenshot-crop_ui-search-string.png'
+
+
+Swiftype is the elastic-based search engine that powers the search in `docs.newrelic.com` and the produdt UI.
+
+As a technical writer at New Relic, you may need to log into Swiftype and do a variety of tasks to make sure your docs are recognized during internal searches.
+
+## Where are Swiftype results used? [#where-used]
+
+A growing number of sites and projects use our Swiftype engine to provide search results:
+
+* The New Relic help center
+* Support landing page
+* Most non-product New Relic sites (developer for related content, docs, storefront, etc.)
+
+## Swiftype crawler info [#crawler-info]
+
+The crawler says it crawls every 12 hours. The rate at which it crawls is relatively slow, which means it could take longer than that to pick up page changes. It also uses some combination of `sitemap.xml` and link collection when crawling.
+
+We can trigger a manual crawl once every 12 hours, if needed. Or, we can have Swiftype crawl specific pages when needed.
+
+The average re-index time for docs is four days.
+
+## Manually add new page to Swiftype index [#manually-index]
+
+You may want to manually add a new page to the Swiftype index when the crawl is too slow and you want the page available right away. Keep in mind that updating the idex may produce quick results for the docs site, you may experience a lag for the docs in the product UI.
+
+To manually add a new page in Swiftype:
+
+1. Ensure the page you want indexed does not have meta robots noindex set and that it's available publicly.
+2. Go to **New Relic > Content**.
+3. In the text input box that says **Filter pages**, paste the URL of the page you want indexed.
+4. If the page is not in the index, Swiftype will provide the option to **Add this URL**. Click that option to add the page to the index.
+5. Insert the URL, making sure to do the following:
+    * Use the entire URL like so: `https://docs.newrelic.com/docs/path/to/doc/`
+    * Ensure the URL has a trailing slash
+
+Swiftype should index it within a minute or so. To check if it's indexed, refresh the page and paste the URL. Note that sometimes the admin UI is cached or lags. You can ask a colleague to check, use the REST API to check, clear the cache, or do a hard refresh of the page and check again.
```

Comment:
should we define "hard refresh"?

Reply:
@stumblefiend I dropped that term because it's confusing. I changed it to this: "...clear the cache, or reload the page and check again."